spec/services/appeal_decision_tree/challenged_decision_status_spec.rb

```diff
@@ -17,7 +24,23 @@
     context 'and the status is other than `pending`' do
       let(:challenged_decision_status) { ChallengedDecisionStatus::RECEIVED }
 
-      it { is_expected.to have_destination(:dispute_type, :edit) }
+      context 'for a case with disputes' do
+        let(:case_type) { CaseType::BINGO_DUTY }
```

Comment:
Like we do e.g. here: https://github.com/ministryofjustice/tax-tribunals-datacapture/blob/master/spec/services/appeal_decision_tree/dispute_type_step_spec.rb#L32